apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.ClientApp/Web.config

```diff
@@ -1,98 +1,76 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <!--
   For more information on how to configure your ASP.NET application, please visit
   https://go.microsoft.com/fwlink/?LinkId=301879
   -->
 <configuration>
-  <appSettings></appSettings>
+  <appSettings />
+  <!--
+    For a description of web.config changes see http://go.microsoft.com/fwlink/?LinkId=235367.
+
+    The following attributes can be set on the <httpRuntime> tag.
+      <system.Web>
+        <httpRuntime targetFramework="4.5.2" />
+      </system.Web>
+  -->
   <system.web>
-    <compilation debug="true" targetFramework="4.6.1"/>
-    <httpRuntime targetFramework="4.6.1"/>
+    <compilation debug="true" targetFramework="4.5.2" />
+    <httpRuntime targetFramework="4.5.2" />
   </system.web>
-  <system.webServer>
-    <handlers>
-      <remove name="ExtensionlessUrlHandler-Integrated-4.0"/>
-      <remove name="OPTIONSVerbHandler"/>
-      <remove name="TRACEVerbHandler"/>
-      <add name="ExtensionlessUrlHandler-Integrated-4.0" path="*." verb="*" type="System.Web.Handlers.TransferRequestHandler" preCondition="integratedMode,runtimeVersionv4.0"/>
-    </handlers>
-  </system.webServer>
+  
   <runtime>
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+       <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
Nit: it would be good to reformat all of this file.

Reply:
Done